gravitee-apim-console-webui/src/organization/configuration/user/new/org-settings-new-user.component.html

```diff
@@ -87,14 +87,14 @@ <h2>Service Account</h2>
 
       <mat-form-field class="org-settings-new-user__card__form-field" appearance="fill">
         <mat-label>Service Name</mat-label>
-        <input matInput formControlName="lastName" required />
+        <input matInput formControlName="lastName" required data-testid="lastName" />
         <mat-hint>Choose a meaningful name for your service account.</mat-hint>
         <mat-error *ngIf="userForm.get('lastName').errors?.required">This field is required.</mat-error>
       </mat-form-field>
 
       <mat-form-field class="org-settings-new-user__card__form-field" appearance="fill">
         <mat-label>Email</mat-label>
-        <input matInput formControlName="email" type="email" />
+        <input matInput formControlName="email" type="email" data-testid="email" />
```

Comment:
We will keep this in the version built for production?

Reply:
As discussed we will merge it like this and see if we need to revise the decision in a few months